packages/compiler/core/types.ts

```diff
@@ -1916,7 +1915,9 @@ export interface EmitContext<TOptions extends object = Record<string, never>> {
    *
    * @param TypeEmitterClass The TypeEmitter to construct your emitted output
    */
-  getAssetEmitter<T>(TypeEmitterClass: typeof TypeEmitter<T, TOptions>): AssetEmitter<T, TOptions>;
+  getAssetEmitter<T>(
+    TypeEmitterClass: any /** typeof TypeEmitter<T, TOptions> https://github.com/TypeStrong/typedoc/issues/2220*/
```

Comment:
Curious if /** @ignore */ works here too?

Reply:
ok everything was updated and it is back to working so removed the workaround.